tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileMain.scala

```diff
@@ -154,12 +158,29 @@ object ProfileMain extends Logging {
       healthCheck.listRemovedExecutors()
       healthCheck.listPossibleUnsupportedSQLPlan()
 
+      if (appArgs.generateDot()) {
+        if (appArgs.compare()) {
+          logWarning("Dot graph does not compare apps")
```

Comment:
just printing but still running it,  skip the generation in this case, we could also add validate in the ProfileArgs to just error out if user specifies

Reply:
I did this change because I wanted to have it run.  I don't want to have to run the profiling tool twice, once to get the dot graphs and once to do the comparison. The warning is just there because some people might think that if they are doing a comparison it might try to do what the original dot code was trying to do.

Reply:
ok, makes sense